x-pack/plugins/secops/public/components/flyout/index.test.tsx

```diff
@@ -234,7 +235,7 @@ describe('Flyout', () => {
       const stateShowIsTrue = set('local.timeline.timelineById.test.show', true, state);
       const storeShowIsTrue = createStore(stateShowIsTrue);
 
-      const showTimeline = jest.fn();
+      const showTimeline = (jest.fn() as unknown) as ActionCreator<{ id: string; show: boolean }>;
```

Comment:
note: with even stricter types using ActionCreator, `jest.fn()` needs me to use this ugly casting to work. If there's another way I would like to know.